data-raw/usual_endpoints.csv

```diff
@@ -1,6 +1,6 @@
-name,url
-wikidata,https://query.wikidata.org/
-dbpedia,https://dbpedia.org/sparql
-databnf,https://data.bnf.fr/sparql
-isidore,https://isidore.science/sparql
-hal,http://sparql.archives-ouvertes.fr/sparql
+name,url,label_property
```

Comment:
@lvaudor should some of this be rdfs:name instead?

Reply:
no I think we're good